st2client/st2client/utils/terminal.py

```diff
@@ -14,32 +14,57 @@
 # limitations under the License.
 
 from __future__ import absolute_import
+
 import os
 import struct
 import subprocess
 import sys
 
 from st2client.utils.color import format_status
 
+DEFAULT_TERMINAL_SIZE_LINES = 80
+DEFAULT_TERMINAL_SIZE_COLUMNS = 150
+
 __all__ = [
     'get_terminal_size'
 ]
 
 
-def get_terminal_size(default=(80, 20)):
+def get_terminal_size(default=(DEFAULT_TERMINAL_SIZE_LINES, DEFAULT_TERMINAL_SIZE_COLUMNS)):
     """
+    Try to retrieve a default terminal size using various system specific approaches.
+
+    If terminal size can't be retrieved, default value is returned.
+
+    NOTE: LINES and COLUMNS environment variables are checked first, if those values are not set /
+    available, other methods are tried.
+
     :return: (lines, cols)
     """
+    # Try LINES and COLUMNS environment variables first like in upstream Python 3 method -
+    # https://github.com/python/cpython/blob/master/Lib/shutil.py#L1203
+    # This way it's consistent with upstream implementation. In the past, our implementation
+    # checked those variables at the end as a fall back.
+    try:
+        lines = os.environ['LINES']
+        columns = os.environ['COLUMNS']
```

Comment:
:+1: on approach, but I think original implementation is safer on corner cases:
https://github.com/python/cpython/blob/master/Lib/shutil.py#L1223-L1231

For example, when only `LINES` or only `COLUMNS` can be overridden by the user.

Maybe we can directly use `shutil`?

Reply:
Yeah, agreed - our implementation requires both variables to be set (but we actually only use columns and ignore lines all together).

Original implementation allows you to specify those independently. I was thinking of making that change as well, but it's a lot more involved than just doing what I do right now (and previous code also requires both to be specified).